src/components/LocationSelector/LocationSelector.tsx

```diff
@@ -1,10 +1,24 @@
 import Image from 'next/image';
 import Link from 'next/link';
-import { ReactElement } from 'react';
+import { ReactElement, useState } from 'react';
 import { RussianCitiesSelect } from './Russia/RussianCitiesSelect';
-import { RussianLocationScroll } from './Russia/RussianLocationScroll';
+import { CISCountriesSelect } from './CIS/CISCountriesSelect';
+
+type ClassesVariant = 'activeClass' | 'onHover';
 
 export const LocationSelector = (): ReactElement => {
+  const classes: Record<ClassesVariant, string> = {
+    activeClass: 'active drop-shadow-[1px_1px_0_orange]',
+    onHover:
+      'hover:transition-filter hover:duration-200 hover:ease hover:opacity-100 hover:drop-shadow-[1px_1px_0_orange]',
+  };
+  const [active, setActive] = useState('1');
```

Comment:
who is active? Need more descriptive variable name

Reply:
use enum when initializing useState, your teammates won't understand what is "1" and "2"

Reply:
gave descriptive names like CIS, Russia